Tests/secrets_white_list.json

```diff
@@ -646,7 +650,11 @@
       "sn1pr11mb0638d849680e1325bc4f7951cc970@sn1pr11mb0638.namprd11.prod.outlook.com",
       "idov@demisto.com",
       "bar.hochman@demisto.com",
-      "powershellteam@hotmail.com"
+      "powershellteam@hotmail.com",
+      "example3@gmail.com",
+      "example1@gmail.com",
+      "jeffrey.collings@gmail.com",
+      "example2@gmail.com"
```

Comment:
???

Reply:
Do not add irrelevant emails that are not supposed to be exposed on public repo please.

Reply:
done